m2cgen/interpreters/java/linear_algebra.java

```diff
@@ -1,19 +1,14 @@
 public static double[] addVectors(double[] v1, double[] v2) {
     double[] result = new double[v1.length];
-
```

Comment:
Removed blank lines to pass API test because they were removed from code generator.